magma/bin/healthcheck

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+HOLD_FILE=$(grep hold_file config.yml | awk '{ print $2 }')
```

Comment:
this is pretty crummy, not sure what is better

Reply:
Could use a yaml parser like [yq](https://github.com/mikefarah/yq) instead. The `hold_file` config change is missing?

Reply:
Err yeah, I left it out of config.yml.template because I  figured it would only be used in production. It just looks like this:
```
  hold_file: tmp/puma.hold
  hold_interval: 10
```

Thought about using yq but I figured the healthcheck should be very fast and wasn't sure about yq's overhead.